versioned_docs/version-2022sp/lecture1.md

```diff
@@ -3,4 +3,532 @@ id: lecture1
 title: Lecture 1
 ---
 
-To be added
+[Lecture Slides](https://docs.google.com/presentation/d/14LCzgpdo6381cS3gfda4fxdumJyl0WS2tRW5PY4EqGg/edit?usp=sharing)
```

Comment:
That's the private folder, I made a public version here: https://docs.google.com/presentation/d/1NX7v1mRL9sih1P6lujQnDVvjNYPcDgiE_XXabcLy6fc/edit?usp=sharing